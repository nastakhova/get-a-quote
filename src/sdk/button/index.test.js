import React from 'react';
import renderer from 'react-test-renderer';

import Button from './index';

it('renders correctly when there are no props', () => {
  const tree = renderer.create(
    <Button />
  	).toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders correctly when have no text', () => {
  const tree = renderer.create(
  	<Button type="edit" />
  	).toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders correctly when have no type', () => {
  const tree = renderer.create(
    <Button text="edit" />
    ).toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders correctly when have type and text', () => {
  const tree = renderer.create(
    <Button text="Edit Trattat" type="edit" />
    ).toJSON();
  expect(tree).toMatchSnapshot();
});

