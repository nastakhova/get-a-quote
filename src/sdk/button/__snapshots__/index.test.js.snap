// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when have no text 1`] = `
<button
  className="button button--edit"
>
  edit
</button>
`;

exports[`renders correctly when have no type 1`] = `
<button
  className="button button--default"
>
  edit
</button>
`;

exports[`renders correctly when have type and text 1`] = `
<button
  className="button button--edit"
>
  Edit Trattat
</button>
`;

exports[`renders correctly when there are no props 1`] = `
<button
  className="button button--default"
>
  button
</button>
`;
